arches/app/templates/base-root.htm

```diff
@@ -46,23 +47,24 @@
             <link rel="stylesheet" href="{% webpack_static 'node_modules/font-awesome/css/font-awesome.min.css' %}" />
         {% endblock css %}
 </head>
+{% endblock head %}
 
+{% block body_content %}
 <body>
-    {% block main_content %}
-        <div id="plugin-mounting-point"></div>
-    {% endblock main_content %}
+    {% block body %}
+    {% endblock body %}
 </body>
+{% endblock body_content %}
 
 {% block javascript %}
     <script src="{% webpack_static 'node_modules/requirejs/require.js' %}"></script>
 
     {% block pre_require_js %}
-        <div
-            id="pluginData"
-            style="display: none;"
-            pluginData='{{ plugin_json }}'
-        ></div>
     {% endblock pre_require_js %}
+
+    {% block arches_modules %}
+        {% include "arches_urls.htm" %}
+    {% endblock arches_modules %}
```

Comment:
while we're here... what if we just have` updateproject.py` create a `project_urls.htm` in the project that gets called after `{% include "arches_urls.htm" %}` so that the project doesn't have to override this file (unless they actually want to explicitly override pre-existing urls).

Reply:
@jacobtylerwalls nice 🎣 !

@whatisgalen I like the idea but it doesn't hold up as a pattern. Since `base-root` lives in Arches core, we'd need to include a `project_urls.htm` in arches-core as well, or else templates would fail to build when running project-less arches-core or tests.